### PR TITLE
support marlin in auto_round format

### PR DESCRIPTION
**Pull request by wenhuach21**

### CODE REVIEW

**Review by yintong-lu — APPROVED** [review]

File: auto_round/export/export_to_autoround/export.py [hunk 1]
```diff
@@ -39,112 +41,122 @@ def check_neq_config(config, data_type, bits, group_size, sym):
...
+        disable_exllama=disable_exllamav1,
+        disable_exllamav2=disable_exllamav2,
+        use_qigen=use_qigen,
+        disable_marlin=disable_marlin,
+    )
+    return QuantLinear
+
+
+def dynamic_QuantLienar_for_packing(backend, bits, group_size, sym):
+    if "auto_round" in backend:
```
Comment (WeiweiZhang1) [inline auto_round/export/export_to_autoround/export.py:123]: autoround and auto_round are easily confused, better use replace() to remove the underscore character before if

File: auto_round/utils.py [hunk 2]
```diff
@@ -583,37 +586,46 @@ def is_autoround_exllamav2_available():
...
     disable_marlin = True
     use_qigen = False
-    if backend == "gptq:qigen":
+    if "qigen" in backend:
         use_qigen = True
-    if backend == "gptq:triton":  ##TODO refine the code
+    elif "triton" in backend:
         use_triton = True
-    if backend == "gptq:marlin":
+    elif "marlin" in backend:
```
Comment (WeiweiZhang1) [inline auto_round/utils.py:599]: asym is not supported for marlin in infer_linear situation, right?

**Review by WeiweiZhang1 — APPROVED** [review]

File: auto_round/utils.py (hunk 2 above)
Comment (wenhuach21) [reply auto_round/utils.py:599]: yes

File: auto_round/export/export_to_autoround/export.py (hunk 1 above)
Comment (wenhuach21) [reply auto_round/export/export_to_autoround/export.py:123]: we will use auto_round as the only one in the futhure

**Comment by wenhuach21** [comment]
regression in this pr

**Pull request merged**